seed/dns.go

```diff
@@ -125,9 +128,10 @@ func (ds *DnsServer) locateChainView(subdomain string) *ChainView {
 func (ds *DnsServer) handleAAAAQuery(request *dns.Msg, response *dns.Msg,
 	subDomain string) {
 
+	log.Debugf("Handling AAAA query")
 	chainView, ok := ds.chainViews[subDomain]
 	if !ok {
-		//log.Errorf("no chain view found for %v", subDomain)
+		log.Errorf("no chain view found for %v", subDomain)
```

Comment:
Same here, was removed as what somewhat spammy, as other chains were attempting to just re-use our current Bitcoin-only resolver.

Reply:
Removed

Reply:
removed